pkg/types/types.go

```diff
@@ -261,10 +265,28 @@ func MergeMetrics(metrics [][]Metric, replicaMismatchConfig cfg.RenderReplicaMis
 			zap.Int("replica_points_total", metricsStat.DataPointCount),
 		)
 	}
+	if len(diffResolutionMetrics) > 0 {
+		logger.Info("metric with different schema observed",
+			zap.Any("schema_mismatch_metrics", diffResolutionMetrics),
```

Comment:
I think we can use zap.Strings here which is better.

Reply:
changed, thanks